newIDE/app/src/EventsSheet/ParameterFields/LeaderboardIdField.js

```diff
@@ -46,103 +49,135 @@ const useFetchLeaderboards = () => {
   return leaderboards;
 };
 
-export function LeaderboardIdField(props: ParameterFieldProps) {
-  const leaderboards = useFetchLeaderboards();
-  const [isAdminOpen, setIsAdminOpen] = React.useState(false);
+export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
+  function LeaderboardIdField(props, ref) {
+    const leaderboards = useFetchLeaderboards();
+    const [isAdminOpen, setIsAdminOpen] = React.useState(false);
+    const genericExpressionFieldRef = React.useRef<?GenericExpressionField>(
+      null
+    );
+    React.useImperativeHandle(ref, () => ({
+      focus: () => {
+        if (genericExpressionFieldRef.current) {
```

Comment:
Can we have a ref on SelectField?

Reply:
I didn't think it was useful but yes it is when using keyboard, will change it